h/models.py

```diff
@@ -115,13 +115,25 @@ def __acl__(self):
     }
     __analysis__ = {
         'filter': {
-            'uri': {
+            'uri_index': {
                 'type': 'pattern_capture',
                 'preserve_original': '1',
                 'patterns': [
                     '([^\\/\\?\\#\\.]+)',
                     '([a-zA-Z0-9]+)(?:\\.([a-zA-Z0-9]+))*',
                     '([a-zA-Z0-9-]+)(?:\\.([a-zA-Z0-9-]+))*',
+                    '://(.+)',
+                    '://(.+)\\?',
+                    '://(.+)\\#'
+                ]
+            },
+            'uri_search': {
+                'type': 'pattern_capture',
+                'preserve_original': '1',
+                'patterns': [
+                    '://(.+)',
+                    '://(.+)\\?',
```

Comment:
Yes and no at the same times. Yes, this can produce loading orphan annotation but at the same time we should treat
```http://site.com/content?mobile=0``` and ```http://site.com/content?mobile=1``` the same in the sake of annotations.